### DISCUSSION TITLE
Is library.kiwix.org blocked in Russia?

### DISCUSSION THREAD

**Discussion by drybin24** [kiwix/kiwix-android#3875] [General]
![Screenshot_2024-06-09-13-35-26-57.jpg](https://github.com/kiwix/kiwix-android/assets/74720460/3898e46e-8155-40a5-86f1-357bf8782958)

It works through Tor, though:

![Screenshot_2024-06-09-13-43-41-91.jpg](https://github.com/kiwix/kiwix-android/assets/74720460/a29f21fc-2bde-4d94-ac65-8dfe2b8494f9)

**Comment by kelson42** [comment]
This would be new. What about https://library.kiwix.org? Does the DNS resolves at all?

**Reply by drybin24** [reply]
do you mean that?



![Screenshot_2024-06-09-13-46-20-51.jpg](https://github.com/kiwix/kiwix-android/assets/74720460/87d275df-c624-4b87-b4c4-d1a80d26c311)

**Reply by drybin24** [reply]
it doesn't work in any browser except for Tor

**Reply by drybin24** [reply]
yeah, my DNS is broken, sorry for bothering
